content/topics/intro-to-relational-databases/_index.md

```diff
@@ -1,15 +1,99 @@
+
 ---
-title: SQL
-ready: True
+title: Introduction to relational database
+ready: true
 ---
 
-## Tutorial
 
-Complete the DataCamp Intro and Intermediate SQL courses:
+### INTRODUCTION TO RELATIONAL DATABASE (SQL)
```

Comment:
I do not understand why this topic is in this branch. Doesn't this need it's own branch.